src/Adapters/Auth/twitter.js

```diff
@@ -5,6 +5,9 @@ var logger = require('../../logger').default;
 
 // Returns a promise that fulfills iff this user id is valid.
 function validateAuthData(authData, options) {
+  if(!options) {
+    throw new Parse.Error(Parse.Error.OBJECT_NOT_FOUND, 'Twitter auth configuration missing');
```

Comment:
same here, it should probably be a -1 as it's a server misconfiguration. easy to spot with the message.